victoriametrics.yaml

```diff
@@ -6,6 +6,13 @@ package:
   copyright:
     - license: Apache-2.0
 
+data:
+  - name: victoria-packages
+    items:
+      victoria-metrics: VictoriaMetrics is a fast, cost-effective and scalable monitoring solution and time series database.
```

Comment:
I think we should retain this in the main package so that we don't end up with victoriametrics-victoria-metrics